charts/pelorus/charts/exporters/templates/_deploymentconfig.yaml

```diff
@@ -32,6 +32,12 @@ spec:
           configMap:
             name: {{ $config.map_name }}
         {{- end }}
+        {{- if .custom_ca_configmap }}
```

Comment:
I think this will not work without updating CRD. It will require re-generating pelorus-operator and creating new release:

https://pelorus.readthedocs.io/en/latest/Development/#versioning-process

Reply:
Would be also great to add hashed example of how this works in the values:

https://github.com/dora-metrics/pelorus/blob/master/charts/pelorus/values.yaml

Reply:
Added a hashed example to the default values file. I also managed to bump the release to 2.0.13-rc.1.

Reply:
> I think this will not work without updating CRD. It will require re-generating pelorus-operator and creating new release:
> 
> https://pelorus.readthedocs.io/en/latest/Development/#versioning-process

I managed to bump the version.